### ISSUE TITLE
Add music prefetch

### ISSUE THREAD

**Issue by SamVerschueren** [CarmenPopoviciu/ngDaftPunk#1]
Always happy to send you a PR if you want :).

**Comment by beeman** [comment]
Great project @CarmenPopoviciu . 

Prefetching is indeed essential when running from the web: https://beeman.github.io/ngDaftPunk/

**Comment by CarmenPopoviciu** [comment]
indeed! I didn't get to it, but I think this is a good small fix to put under the Christmas tree ;). I'm going to fix this asap
